src/coreclr/nativeaot/Directory.Build.props

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <EnableDefaultItems>false</EnableDefaultItems>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
```

Comment:
Why?

EnableDefaultItems being off gives a much better error when things get out of state, and it being on leads to confusion when some particles get created (e.g. `copy Foo.cs Foo.bak.cs`... now the project fails)

I really really really (etc) do not want us to ever turn this on.

So: Why turn it on?